### PR TITLE
adding new "resources" page

### PR DESCRIPTION
**Pull request by slugmann321**
## Summary
Adding a new page called "resources and further reading" inside of the section "further reading". 

This is based on Oisin's suggestion for an "awesome" Obol list for devs. 
Once this is created, I will create a shortened, pretty version for the homepage of the Obol Network github page.

### CODE REVIEW

File: docs/fr/resources.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+sidebar_position: 6
+description: Resources for further reading
```
Comment (OisinKyne) [inline docs/fr/resources.md:3]:
```suggestion
description: A collection of links to products and content relating to Distributed Validators.
```

File: docs/fr/resources.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+---
+
+# Resources and Further Reading
+See here a list of internal and external resources for further reading and exploration.
```
Comment (OisinKyne) [inline docs/fr/resources.md:7]:
```suggestion
# Useful Links
The following is a curated list of the best internal and external resources for using, creating, running, building, and researching Distributed Validators. To add to this list, please open a [pull request](https://github.com/ObolNetwork/obol-docs/pulls/).
```

**Review by OisinKyne — APPROVED** [review]
Slight copy tweaks, can commit suggestions and then merge.

File: docs/fr/resources.md (hunk 1 above)
Comment (maeliosakyne) [reply docs/fr/resources.md:3]: in agreement with oisins suggestion

File: docs/fr/resources.md (hunk 2 above)
Comment (maeliosakyne) [reply docs/fr/resources.md:7]: agree with this suggestion

**Comment by maeliosakyne** [comment]
i agree with oisins suggestions, but can't actually 'approve' this as it has already been merged

**Comment by slugmann321** [comment]
Yes I didn't realize I had to press "commit" (I thought his changes would automatically be implemented) but I made his proposed changes in a separate PR:
https://github.com/ObolNetwork/obol-docs/pull/411

**Pull request merged**